README.md

```diff
@@ -46,7 +46,17 @@ We will focus on building an open API first. Then developers can use the API to
 
 ## User stories so far
 
-Our goal is to keep things simple and not reinvent wheels. So far we have only two user roles: participants and chapter organizers.
+Our goal is to keep things simple and not reinvent wheels. So far we have only two user roles: participants and chapter organizers, both of which are users.
+
+### As a user
+
+- I can open a registration page where I can sign up with email password.
+
+- I can log in with my email/password or I can log in with social login via Github.
```

Comment:
User can login via email/password, Github, Facebook, or Google account

Reply:
we decided to use Google only for now and once that is implemented and working we can add the others